lib/migration.pm

```diff
@@ -151,7 +151,8 @@ sub remove_dropped_modules_packages {
 sub disable_installation_repos {
     if (is_s390x) {
         my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}'");
```

Comment:
Just add the "| tr '\n' ' '" at the end this cmd line then got the output separate with space, you don't need change other code then.

Reply:
VR: https://openqa.suse.de/tests/14809451#